src/status_im/transport/inbox.cljs

```diff
@@ -0,0 +1,191 @@
+(ns status-im.transport.inbox
+  (:require [re-frame.core :as re-frame]
+            [status-im.native-module.core :as status]
+            [status-im.utils.handlers :as handlers]
+            [status-im.transport.utils :as web3.utils]
+            [taoensso.timbre :as log]))
+
+(defn- parse-json
+  ;; NOTE(dmitryn) Expects JSON response like:
+  ;; {"error": "msg"} or {"result": true}
+  [s]
+  (try
+    (let [res (-> s
+                  js/JSON.parse
+                  (js->clj :keywordize-keys true))]
+      ;; NOTE(dmitryn): AddPeer() may return {"error": ""}
+      ;; assuming empty error is a success response
+      ;; by transforming {"error": ""} to {:result true}
+      (if (and (:error res)
+               (= (:error res) ""))
+        {:result true}
+        res))
+    (catch :default e
+      {:error (.-message e)})))
+
+(defn- response-handler [error-fn success-fn]
+  (fn handle-response
+    ([response]
+     (let [{:keys [error result]} (parse-json response)]
+       (handle-response error result)))
+    ([error result]
+     (if error
+       (error-fn error)
+       (success-fn result)))))
+
+(defn add-peer [enode success-fn error-fn]
+  (status/add-peer enode (response-handler error-fn success-fn)))
+
+(defn fetch-peers
```

Comment:
Move comment into docstring

Reply:
there is no docstring in register-handler-fx it's a function call